src/DependencyInjection/Configuration/Configuration.php

```diff
@@ -127,7 +128,21 @@ private function addClientsSection(ArrayNodeDefinition $rootNode): void
                                     ->scalarNode('serialization')->defaultValue('default')->end()
                                     ->scalarNode('cluster')->defaultNull()->end()
                                     ->scalarNode('prefix')->defaultNull()->end()
-                                    ->enumNode('replication')->values([true, 'sentinel'])->end()
+                                    ->enumNode('replication')
+                                        ->values([true, 'predis', 'sentinel'])
+                                        ->beforeNormalization()
+                                            ->ifTrue(static fn ($v) => $v === true)
+                                            ->then(static function () {
+                                                trigger_deprecation(
+                                                    'snc/redis-bundle',
+                                                    '4.6',
```

Comment:
4.5.1 ?

Reply:
Ah yeah sorry my oversight, should have tagged 4.6. Anyways I'll leave this like this now because it's not a big issue